devices/ePaper/version.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
you are missing the category.txt file. Please check other displays and use the same ;-)